src/main/java/flight/reservation/plane/Aircraft.java

```diff
@@ -0,0 +1,37 @@
+package flight.reservation.plane;
+
+public abstract class Aircraft {
+	protected String model;
+	protected int passengerCapacity;
+	protected int crewCapacity;
+
+	public Aircraft(String model, int passengerCapacity, int crewCapacity) {
+		this.model = model;
+		this.passengerCapacity = passengerCapacity;
+		this.crewCapacity = crewCapacity;
+	}
+
+	public String getModel() {
+		return model;
+	}
+
```

Comment:
Could the setters be removed, since they should not get used and the variables shouldn't change?